Importer/CsvImporter.php

```diff
@@ -193,9 +173,9 @@ protected function importData(array $parsedData, $dataIsAssociativeArray, $flush
 
         try {
             $this->entityManager->flush();
-        } catch (ORMException $exception) {
-            $message = 'Error when flushing for entity.';
-            $this->handleImporterError(new Error($message, $exception));
+        } catch (\Exception $exception) {
+            $error = new Error('Error during flush', $exception);
```

Comment:
Symfony handles exception by dispatching `KernelEvents::Exception` event. You can check:

https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpKernel/HttpKernel.php#L79